README.md

```diff
@@ -183,7 +183,10 @@ $client->createLink($link);
 To update an already existing link you need to pass the url of the link and the updated `Link` instance to the  `updateLink` method
 
 ```
-$client->updateLink($url, $data, $type);
```

Comment:
This is the old method signature. I also thought that pulling together a plausible string of calls would be helpful here.

Reply:
Thank you for patching the readme!